cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -316,10 +316,8 @@ func getControllerManagerCommand(cfg *kubeadmapi.ClusterConfiguration) []string
 	// Each node will get a subspace of the address CIDR provided with --pod-network-cidr.
 	if cfg.Networking.PodSubnet != "" {
 		// TODO(Arvinderpal): Needs to be fixed once PR #73977 lands. Should be a list of maskSizes.
```

Comment:
@tedyu please move the comment too.

@Arvinderpal what is the state of this TODO?
https://github.com/kubernetes/kubernetes/pull/73977 still shows a single max size:
https://github.com/kubernetes/kubernetes/pull/73977/files#diff-6bbac02bd63acca6e05007f7c054396cR84

Reply:
@neolit123 
Your comment has been addressed.